src/main/java/seedu/address/model/client/ClientProjectList.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.project.Project;
+
+
+/**
+ * Represents the list of projects the client is associated with.
+ */
+public class ClientProjectList {
+
+    private ArrayList<Project> projects;
+
+    /**
+     * Constructs an empty project list.
+     */
+    public ClientProjectList() {
+        this.projects = new ArrayList<Project>();
+    }
+
+    /**
+     * Adds a project to the list.
+     * @param project Project to be added.
+     */
+    public void addProject(Project project) {
+        requireNonNull(project);
+        this.projects.add(project);
+    }
+
+    /**
+     * Removes a project from the list.
+     * @param project Project to be removed.
+     */
+    public void removeProject(Project project) {
+        this.projects.remove(project);
+    }
+
+    /**
+     * Returns a list of all projects.
+     * @return String representing all projects in the list
+     */
+    public String listAllProjects() {
+        if (projects.size() > 0) {
+            String list = this.projects.toArray().toString();
+            return list;
+        } else {
+            return "none";
```

Comment:
Try to avoid hardcoding strings in. Method may be removed (unless you intend it to use it somewhere)

Reply:
As of now plan to use it, but noted and will remove if it is unused